doc/content/devices/best-practices/_index.md

```diff
@@ -102,3 +102,13 @@ In the case of link loss, do the following:
 - Decrease data rate step by step, and try again
 - Reset to default channels, and try again
 - Send periodic join requests with backoff
+
+## Support force rejoin command or downlink
+
+The Force Rejoin Command (`ForceRejoinReq`) is part of the LoRaWAN specifications versions 1.1. When using the command, the device to asked to immediately transmit a Rejoin-Request. Force Rejoin Commands enable, among others, end devices to easily switch LoRaWAN networks by initiating a new session with a new Network Server provider.
```

Comment:
```suggestion
The Force Rejoin Command (`ForceRejoinReq`) is part of the LoRaWAN specifications version 1.1. When using this command, the device immediately transmits a Rejoin Request message. Force Rejoin Commands enable end devices to, among other, easily switch LoRaWAN networks by initiating a new session with a new Network Server provider.
```

Reply:
this wasn't really clear so I'm not sure if my suggestion is hitting the point either. you should rephrase this either way